src/web/recorder/webpack.config.js

```diff
@@ -19,7 +19,13 @@ module.exports = {
     rules: [
       {
         test: /\.(j|t)sx?$/,
-        use: 'ts-loader',
+        loader: 'babel-loader',
+        options: {
+          presets: [
+            "@babel/preset-typescript",
+            "@babel/preset-react"
```

Comment:
How did the recorder before build the React stuff? Since I don't see any React presets in the webpack config from before.

Reply:
typescript has a built in JSX transpiler that ts-loader was used.